### PR TITLE
Change single link and add frozenlist.

### PR DESCRIPTION
**Pull request by UP929312**
## What do these changes do?

Add in missing requirement, fix link that's different for no reason.

These changes fix the missing requirement for frozenlist in the file, as well as changing a different link in a list that should be inline with the others.

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -166,13 +166,15 @@ Requirements
 - charset-normalizer_
 - multidict_
 - yarl_
+- fozenlist_
 
 Optionally you may install the cChardet_ and aiodns_ libraries (highly
 recommended for sake of speed).
 
-.. _charset-normalizer: https://pypi.org/project/charset-normalizer
+.. _charset-normalizer: https://pypi.python.org/project/charset-normalizer
```
Comment (webknjaz) [inline README.rst]: This change is incorrect.

**Review by webknjaz — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (UP929312) [reply README.rst]: Why is it incorrect? It redirects to the regular URL anyway

Comment (webknjaz) [reply README.rst]: The canonical domain is pypi.org.

**Comment by UP929312** [comment]
@webknjaz I fixed the url

**Review by webknjaz — APPROVED** [review]

**Pull request merged**